src/NovelRT/Audio/AudioService.cpp

```diff
@@ -69,10 +70,12 @@ namespace NovelRT::Audio
             return _noBuffer;
         }
 
-        std::vector<uint16_t> data;
+        std::vector<int16_t> data;
```

Comment:
Why `int16_t` for a data buffer?

Reply:
I had switched it because the datatype returned by libsndfile _should_ only be signed. This can be switched if desired however - I believe it'll work with uint16_t as well

Reply:
Generally I use `uint8_t` which is what prompted the question since its just a byte buffer right?

If that's gonna cause breakages then just ignore this is just me being curious

Reply:
Yeah, based on sndfile providing `short` values to `data`, this would need to be `uint16_t`  o_or `int16_t` therwise there's the potential to lose data based on casting (Windows does not like using `uint8_t` here).

Reply:
It sounds like this is sample buffer data,so maybe it should be called something like `sampleData` rather than just `data`.